docs/tasks.rst

```diff
@@ -41,48 +41,48 @@ Options
 ------------------------------------------
 
 
-``-o related_list RELATEDLIST``
+``--related-list RELATEDLIST``
```

Comment:
I didn't realize these didn't get in with the "Normalized Task Options" PR. Thanks for including these.